subprojects/core/src/integTest/groovy/org/gradle/api/tasks/MissingTaskDependenciesIntegrationTest.groovy

```diff
@@ -313,9 +313,10 @@ class MissingTaskDependenciesIntegrationTest extends AbstractIntegrationSpec {
             Consider using Task.dependsOn instead of an input file collection.""".stripIndent())
     }
 
-    void expectMissingDependencyDeprecation(String producer, String consumer) {
+    void expectMissingDependencyDeprecation(String producer, String consumer, File producedConsumedLocation) {
         executer.expectDocumentedDeprecationWarning(
             "Task '${consumer}' uses the output of task '${producer}', without declaring an explicit dependency (using Task.dependsOn() or Task.mustRunAfter()) or an implicit dependency (declaring task '${producer}' as an input). " +
+                "The location which is an input/output is '${producedConsumedLocation.absolutePath}'. " +
```

Comment:
How about:

```suggestion
                "The consumed output is at '${producedConsumedLocation.absolutePath}'. " +
```

Reply:
We also don't say anything about consumer/producer above, so I would not introduce a new term in this part of the message.

How about 
```
The used output is at ...
```

Reply:
What about something like:

> Gradle detected a problem with at following location: ...."
>     Task .... uses the output of task ... without ...